examples/test.rs

```diff
@@ -38,6 +39,33 @@ use camera_controllers::{
 
 use gfx::traits::*;
 
+struct WindowOuput<R: gfx::Resources> {
```

Comment:
note: this is hopefully only a temporary measure until we figure out how to get the piston window libs to play nicely with gfx